vmdk_plugin/Makefile

```diff
@@ -88,6 +88,14 @@ VMDKOPS_MODULE_SRC = $(VMDKOPS_MODULE)/*.go $(VMCI_SRC)
 SRC = plugin.go main.go log_formatter.go refcnt.go \
 	utils/fs/fs.go utils/config/config.go
 
+# Canned recipe
+define log_target
+@echo
+@echo "=> Running target $@"
```

Comment:
Time message makes it harder to grep if time is at the end, also reading wise looks cleaner (to my eyes), open to change but can do it separately.